src/guide_curves/CCPACSGuideCurves.cpp

```diff
@@ -101,12 +101,14 @@ std::vector<double> CCPACSGuideCurves::GetRelativeCircumferenceParameters() cons
         const CCPACSGuideCurve* root = GetGuideCurve(iguide).GetRootCurve();
         relCircs.push_back(*root->GetFromRelativeCircumference_choice2());
     }
-    if ( relCircs.back() < 1.0 ) {
-        relCircs.push_back(1.0);
-    }
 
     std::sort(relCircs.begin(), relCircs.end());
 
+    double eps = 1e-10;
+    if ( relCircs.back() < 1.0 - eps ) {
+        relCircs.push_back(1.0);
+    }
+
```

Comment:
This can be changed, by using the contains() function. On the other hand, as @joergbrech pointed out, CCPACSWing and CCPACSGuideCurves have both almost the same code. There should only be one place for it.

In principle, this should stay in guidecurves, as it can be used then by all other classes (fuselage etc) that use guide curves.